pinot-query-planner/src/main/java/org/apache/pinot/query/planner/stage/TableScanNode.java

```diff
@@ -19,23 +19,25 @@
 package org.apache.pinot.query.planner.stage;
 
 import java.util.List;
-import org.apache.calcite.rel.type.RelDataType;
 import org.apache.pinot.query.planner.serde.ProtoProperties;
+import org.apache.pinot.spi.data.FieldSpec;
 
 
 public class TableScanNode extends AbstractStageNode {
   @ProtoProperties
   private String _tableName;
   @ProtoProperties
   private List<String> _tableScanColumns;
+  @ProtoProperties
+  private FieldSpec.DataType _rowType;
 
   public TableScanNode(int stageId) {
     super(stageId);
   }
 
-  public TableScanNode(int stageId, RelDataType rowType, String tableName, List<String> tableScanColumns) {
+  public TableScanNode(int stageId, FieldSpec.DataType rowType, String tableName, List<String> tableScanColumns) {
```

Comment:
Are we ok with replacing rowType with a columnType ?

Reply:
good catch. actually pinot doens't support nested schema (for now) so I think we can do a list of datatypes

Reply:
2nd thought. no need to encode this here. we will add RexExpression if needed